### DISCUSSION TITLE
Thanks for the 1.7.0 Theming/Configuration Updates

### DISCUSSION THREAD

**Discussion by joshskidmore** [davatorium/rofi#1448] [General]
There have been a couple discussion posts, issues and Reddit posts about how 1.7.0 has broken themes and scripts. I haven't seen any posts showing appreciation for the updates, so I thought I would. 

As a developer that spends a lot of time on the front end, this release has made it so much easier to organize, cascade and structure configurations and themes. I appreciate the effort and thoughtfulness that has went into this release. While there might be some frustration initially, I believe these updates will lead to better portability and sharing of more complex Rofi configurations and themes.

I've been using Rofi for years now, and slowly, I've used to it power many tools that improve my daily workflow, but plucking them out individually (to share) has been a bit complicated. With this release, I plan to share/open source some of these - including my theme and some of my scripts that use Rofi to power the navigation.

Thanks to the team for the continued work to make Rofi such a wonderful tool.

My first share will an almost perfectly replicated Windows 10 Alt-Tab switcher using just the window modi. I've been able to emulate this so well - from the look and feel, to key bindings, to the endless circulating key behavior and the key release to activate the selected window.

**Comment by DaveDavenport** [comment]
Thanks for the positive feedback.  It is good to hear that people see the improvements I am trying make.
(and all the complaints are a bit demotivating, its a hobby project with very little developer time.)
